vendor/github.com/containerd/zfs/go.mod

```diff
@@ -7,4 +7,5 @@ require (
 	github.com/containerd/continuity v0.0.0-20210208174643-50096c924a4e
 	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
 	github.com/pkg/errors v0.9.1
+	gotest.tools/v3 v3.0.3
```

Comment:
Can we refactor the zfs test to not introduce this as an extra dependency? Looking at the test, it's only using some minimal functionality of `gotest.tools`

Reply:
Opened https://github.com/containerd/zfs/pull/40

Reply:
Thanks, updated